pinot-common/src/main/java/org/apache/pinot/common/utils/TlsUtils.java

```diff
@@ -240,10 +230,6 @@ public static void installDefaultSSLSocketFactory(String keyStoreType, String ke
       // HttpsURLConnection
       HttpsURLConnection.setDefaultSSLSocketFactory(sc.getSocketFactory());
 
-      // Apache HTTP client 3.x
-      Protocol.registerProtocol("https",
-          new Protocol(CommonConstants.HTTPS_PROTOCOL, new PinotProtocolSocketFactory(sc.getSocketFactory()), 443));
```

Comment:
@apucher This is the only usage. Do you think we still need it now?

Reply:
if and only if Apache HHTP Client 3.x is still being used by pinot

Reply:
Hmm, in that case should we remove this as the last cleanup step instead of first? We haven't started cleaning up other usages yet

Reply:
Right, let me raised a PR with my other changes.